package.json

```diff
@@ -1,11 +1,11 @@
 {
-  "name": "monaco-components",
```

Comment:
Why was the package name dropped?

Reply:
Not intended. Will bring it back. Interestingly a name is not required for the workspace package.

Reply:
Done ⬇️